modules/subscriber/main.tf

```diff
@@ -1,5 +1,6 @@
 locals {
   has_discovery_rate = var.discovery_rate != ""
+  name_prefix        = "${substr(var.name, 0, 37)}-"
```

Comment:
should we mention in the docs somewhere that roles will have this abbreviated prefix?